guides/common/modules/con_transport-modes-for-remote-execution.adoc

```diff
@@ -21,14 +21,12 @@ ifdef::katello,orcharhino,satellite[]
 * To enable pull mode on an existing host, continue with xref:Configuring_a_Host_to_Use_the_Pull_Client_{context}[].
 endif::[]
 ifdef::managing-hosts[]
-* To migrate a host from Katello Agent, see xref:Migrating_Hosts_From_Katello_Agent_to_Remote_Execution_{context}[].
 * To enable pull mode on a new host, continue with either of the following procedures:
 
 ** xref:Creating_a_Host_{context}[]
 ** xref:Registering_Hosts_by_Using_Global_Registration_{context}[].
 endif::[]
 ifndef::managing-hosts[]
-* To migrate a host from Katello Agent, see {ManagingHostsDocURL}Migrating_Hosts_From_Katello_Agent_to_Remote_Execution_managing-hosts[Migrating from Katello Agent to Remote Execution]  in _{ManagingHostsDocTitle}_.
```

Comment:
This seems relevant for one more release. @jeremylenz how are you thinking about this?

Reply:
I was thinking about it this way: The way we normally approach our documentation is that we don't generally mention _changes_ to the product (other than in release notes); rather, the documentation reads _as if it's always been this way._ So staying consistent with that, we'd remove this section as well. If users have this version of Katello/Satellite installed, they've already migrated to REX and have followed these instructions in previous versions of documentation.

I believe the 6.15 release note should still link to the 6.14 documentation for migrating hosts.